daprdocs/content/en/reference/components-reference/supported-secret-stores/akeyless.md

```diff
@@ -0,0 +1,139 @@
+---
+type: docs
+title: "Akeyless"
+linkTitle: "Akeyless"
+description: Information about the Akeyless secret store component configuration.
+---
+
+## Create the Akeyless component
+
+To setup Akeyless secret store create a component of type `secretstores.akeyless`. See [this guide]({{% ref "setup-secret-store.md#apply-the-configuration" %}}) on how to create and apply a secretstore configuration. See this guide on [referencing secrets]({{% ref component-secrets.md %}}) to retrieve and use the secret with Dapr components.
+
```

Comment:
Can you add a "Component format" section that covers all the potential field for the component. Then follow this with the "Spec metadata fields". The Authentication Methods section can then follow after this as examples of using the component with different auth methods. See example here https://docs.dapr.io/reference/components-reference/supported-secret-stores/aws-secret-manager/